lib/galaxy/managers/workflows.py

```diff
@@ -1059,13 +1079,19 @@ def _workflow_to_dict_run(self, trans, stored, workflow, history=None):
                 }
                 for oc in step.output_connections
             ]
+            annotations_dict = {
+                "annotation": stored.annotations[0].annotation,
+                "update_time": (stored.annotations[0].stored_workflow.update_time).isoformat(),
+                "tags": self.convert_to_dict_from_many(stored.annotations[0].stored_workflow.tags),
```

Comment:
```suggestion
                "tags": self.convert_to_dict_from_many(stored.tags),
```
but also we have better ways to serialize tags